gensim/corpora/bleicorpus.py

```diff
@@ -109,14 +112,25 @@ def save_corpus(fname, corpus, id2word=None, metadata=False):
                 doc = list(doc)
                 offsets.append(fout.tell())
                 parts = ["%i:%g" % p for p in doc if abs(p[1]) > 1e-7]
-                fout.write(utils.to_utf8("%i %s\n" % (len(doc), ' '.join(parts))))
+                fout.write(
+                    utils.to_utf8(
+                        "%i %s\n" %
+                        (len(doc), ' '.join(parts))))
 
-        # write out vocabulary, in a format compatible with Blei's topics.py script
+        # write out vocabulary, in a format compatible with Blei's topics.py
+        # script
         fname_vocab = utils.smart_extension(fname, '.vocab')
-        logger.info("saving vocabulary of %i words to %s" % (num_terms, fname_vocab))
+        logger.info(
```

Comment:
Dtto.

Also, if fixing old code, can you replace `logger` methods which have `method("string" % params)` with `method("string", params)`?